test/unit-tests/components/viewmodels/right_panel/RoomSummaryCardTopicViewModel-test.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2025 New Vector Ltd.
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { renderHook } from "jest-matrix-react";
+import { act, type SyntheticEvent } from "react";
+
+import { useRoomTopicViewModel } from "../../../../../src/components/viewmodels/right_panel/RoomSummaryCardTopicViewModel";
+import { mkStubRoom, stubClient } from "../../../../test-utils";
+import defaultDispatcher from "../../../../../src/dispatcher/dispatcher";
+import { onRoomTopicLinkClick } from "../../../../../src/components/views/elements/RoomTopic";
+
+jest.mock("../../../../../src/components/views/elements/RoomTopic");
+
+describe("RoomSummaryCardTopicViewModel", () => {
+    const client = stubClient();
```

Comment:
If we don't need `MatrixClient.get()` in the children component

```suggestion
    const client = createTestClient();
```